scipy/special/orthogonal.py

```diff
@@ -54,6 +54,8 @@
   la_roots               -- Generalized Laguerre
   h_roots                -- Hermite
   he_roots               -- Hermite (unit-variance)
+  h_roots_asy            -- Hermite, asymptotic version
```

Comment:
Is there a good reason to expose these?  Shouldn't this all just go through `h_roots`?  The method used to produce the roots shouldn't really matter to someone using the function and you changes below already switch methods at a reasonable point.